### PR TITLE
build(sauce/bs): make some browsers required in CI

### PR DESCRIPTION
**Pull request by marclaval**
As discussed with @IgorMinar and @juliemr , let's add the very stable browsers as required jobs in CI.

For other ones, they are kept in optional jobs. Analytics will be added later in order to have more details about them.

### CODE REVIEW

File: browser-providers.conf.js [hunk 1]
```diff
@@ -1,3 +1,31 @@
+// Unique place to configure the browsers which are used in the different CI jobs in Sauce Labs (SL) and BrowserStack (BS).
```
Comment (IgorMinar) [inline browser-providers.conf.js]: does `null` stand for local browser? can you add that to the documentation in a separate PR please?

Comment (IgorMinar) [inline browser-providers.conf.js]: or does it mean that we don't run that browser anywhere?

File: browser-providers.conf.js [hunk 2]
```diff
@@ -1,3 +1,31 @@
...
+// If a category becomes empty (e.g. BS and required), then the corresponding job must be commented out in Travis configuration.
+// TODO(mlaval): iOS9 deactivated as not reliable, reactivate after https://github.com/angular/angular/issues/5408
```
Comment (IgorMinar) [inline browser-providers.conf.js]: move this above the iOS9 line below - in a separate PR. we'll merge this one before you come online.

**Comment by IgorMinar** [comment]
lgtm

**Comment by IgorMinar** [comment]
please fix up the docs in a separate PR

**Comment by mlaval** [comment]
Merge failed, so comments integrated here.

But SauceLabs seems to have issues, the job hangs forever after running all the tests successfully.
Note that it has also been happening for about 15 hours in all builds before this PR.

**Comment by IgorMinar** [comment]
This is weird.. I've restarted the job. if that doesn't help and raw log doesn't reveal anything (sometimes travis UI doesn't show complete log), try to rebase the PR.

**Comment by mlaval** [comment]
Rebasing didn't help, but I found that the culprit was Chrome Canary (i.e. version 49) which was released 5 days ago. A new version of the Selenium drivers might be needed.
Falling back to the beta version (48) made the job be green again.

**Comment by IgorMinar** [comment]
great. let's get this in!

**Pull request closed without merging**